### ISSUE TITLE
Add option to automatically attempt to reconnect after connection drops

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/vscode-ev3dev-browser#55]
If using a flaky communication method (either because of the transit itself or because of CPU saturation dropping connections), it would be nice if the extension would automatically try to reconnect once or twice after losing communications. Alternately, at least prompting the user to try and reconnect would be good.

**Comment by WasabiFan** [comment]
Thinking about this a bit, I think it might make sense to entirely remove the device entry from the browser pane/tree upon disconnection and instead display a notification at the top -- likely with options to reconnect or connect to a different device. It isn't immediately clear with the current UI how you reconnect after the connection drops.

Relatedly, if I am on Wi-Fi and I want to switch to USB, I actually have to choose "connect to a different device" rather than "reconnect". I am not sure how we would fix that problem though.

**Comment by dlech** [comment]
Yeah, this is not an easy one. I'll have to think about it for a while.
